Lib/test/test_descr.py

```diff
@@ -4724,6 +4724,24 @@ def __call__(self, arg):
         self.assertRegex(repr(method),
             r"<bound method qualname of <object object at .*>>")
 
+    def test_bpo25750(self):
+        # bpo-25750: calling a descriptor (implemented as built-in
+        # function with METH_FASTCALL) should not crash CPython if the
+        # descriptor deletes itself from the class.
+        try:
+            from _testcapi import bad_get
+        except ImportError:
+            return
+
+        class Descr:
+            __get__ = bad_get
+        class X:
+            descr = Descr()
+            def __new__(cls):
+                cls.descr = None
+                cls.lst = [2**i for i in range(10000)]
```

Comment:
I'm at work at the moment, so I don't have a chance to test this, but do we need `cls.lst` to reproduce the crash?

Reply:
> do we need cls.lst to reproduce the crash?

That line is just to mess with the memory to ensure a sufficiently corrupted state.